app/scripts/views/ready.js

```diff
@@ -55,13 +58,23 @@ function (Cocktail, BaseView, Template, Session, Xss, Strings,
     },
 
     _createMarketingSnippet: function () {
-      var marketingSnippet = new MarketingSnippet({
+      var marketingSnippet;
+      var marketingSnippetOpts = {
         type: this.type,
         service: this.relier.get('service'),
         language: this.language,
         el: this.$('.marketing-area'),
-        metrics: this.metrics
-      });
+        metrics: this.metrics,
+        able: this._able
+      };
+
+      // Fx iOS scheduled to launch 6/2
+      if (this._able.choose('fxIos').launched) {
```

Comment:
Same [comment from above](https://github.com/mozilla/fxa-content-server/pull/2316/files#r28757810) applies here. Let's make sure this does not explode if the experiment's startDate has not yet passed.